spec/factories/user.rb

```diff
@@ -8,5 +8,24 @@
     security_code 'test'
     security_code_confirmation 'test'
     locale 'en'
+    display_name 'John Doe'
+
+    trait :activated do
+      after(:create) do |u|
```

Comment:
is there a reason we're doing this in an `after(:create)` rather than `after(:build)`? It's nice to be able to create persisted models for testing.